### PR TITLE
Create framework for OSPO Local Meetups

### PR DESCRIPTION
**Pull request by anajsana**
This framework defines the requirements to host and organize OSPO local meetups in local languages powered by TODO Group.

Signed-off-by: Ana Jimenez Santamaria <ana@todogroup.org>

### CODE REVIEW

File: local-meetups/framework.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+3. [Introducing Local OSPO Meetups powered by TODO Group](#introducing-local-ospo-meetups-powered-by-todo-group)
+   + [The Framework](#the-framework)
+   + [Meetup expectations](#meetup-expectations)
+   + [Local Organizer Requirements and Responsibilities](#local-organizer-requirements-and-responsibilities)
+5. [Interested?](#interested)
+
+
+
+## About OSPOlogy 
+OSPOlogy is an initiative under the TODO group that offers best practice-sharing online workshops that are open to all, from folks starting their OSPOs to seasoned OPSOers! These sessions are typically led by a TODO member covering one topic or sharing their organization’s OSPO journey.
```
Comment (geekygirldawn) [inline local-meetups/framework.md:15]: I think we should remove the hyphen between practice and sharing.

File: local-meetups/framework.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+For further details see https://github.com/todogroup/ospology.
+
+## Problem statement
+TODO Group members requested local events to find and network with OSPOers near their area, to different OSPO topics.
+Introducing OSPOlogy Meetups, powered by TODO Group.
+
+We are introducing a new framework for in-person local meetups. OSPO Local Meetups aim to provide a forum for like-minded professionals to share knowledge and experience on OSPOs and Open Source Management within specific regions.
```
Comment (geekygirldawn) [inline local-meetups/framework.md:22]: This is a duplicate of the sentence below in the introducing section.

File: local-meetups/framework.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+In a nutshell, the framework defines the following local meetup requirements:
+
+* Hybrid gatherings that are open to TODO members and the public
+* Hosted by a Local TODO Organizer
+* Follow [Chatham House Rules](https://www.chathamhouse.org/about-us/chatham-house-rule)
+  * Recordings are not allowed
+  * The information received can be shared as meet-up notes, but does not reveal the identity of who said it
+* Provides a forum for like-minded professionals to share knowledge and experience on OSPOs and Open Source Management
+* Encourages vendor-neutraly
```
Comment (geekygirldawn) [inline local-meetups/framework.md:38]: should be neutrality

File: local-meetups/framework.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+* Encourages diversity of attendees and regions to participate including hosting meetups in local languages
+* TODO community participants can propose a new local TODO meetup
+* Needs to have a Local TODO Organizer
```
Comment (geekygirldawn) [inline local-meetups/framework.md:41]: This duplicates the second bullet.

File: local-meetups/framework.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+* Is under Linux Foundation [policies](https://www.linuxfoundation.org/policies/) and [code of conduct](https://events.linuxfoundation.org/about/code-of-conduct/)
+* Participants must register for TODO Local meet-ups via [Linux Foundation’s community platform](https://community.linuxfoundation.org/todo-group-europe/).
+
+### Meetup Expectations
+
+OSPO Local meetups powered by the TODO Group is committed to vendor neutrality and aims to become a forum for like-minded professionals to share knowledge and experiences on OSPOs. Even though we understand that during a conversation, OSPO tooling and/or services examples might arise, **vendor pitches must not be part of the meetup core discussions**.
```
Comment (geekygirldawn) [inline local-meetups/framework.md:47]: Let's add a bit to this. Maybe something like what I proposed in Slack to add to or replace the part currently in bold:
Vendors may not propose solutions or use the meetups under the TODO Group umbrella for any discussions designed to generate sales for a vendor. Meetups are not intended to be vendor pitches and can’t be used for that purpose if you would like to have the meetup under the TODO Group branding.

File: local-meetups/framework.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+**OPTION 2**
+
+* You are an [OSPOlogy Maintainer](https://github.com/todogroup/ospology/blob/main/meetings/GOVERNANCE.md)
+AND
+* Your organization does not derive a significant portion of its revenue (more than 50%) as a solution provider to other OSPOs 
+
+Local TODO Organizer shall be responsible for:
+
+* Managing registration and meetup description via [Linux Foundation’s community platform](https://community.linuxfoundation.org/todo-group-europe/).
+* Providing meet-up location and logistics
```
Comment (geekygirldawn) [inline local-meetups/framework.md:65]: I think we can remove this hyphen between meet and up

File: local-meetups/framework.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+* Reporting to TODO Program Manager and Steering Committee each quarter to track meetup evolution and new topics
+* Posting a recap to [OSPO Forum](https://github.com/todogroup/ospology/discussions/categories/announcements) in English language.
```
Comment (geekygirldawn) [inline local-meetups/framework.md:67]: I'd remove the word language and just say "in English"

File: local-meetups/framework.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+
+
+## Interested?
+
+If you are interested in organizing a TODO Local Meetups in your region feel free to open a new [GH Discussion under OSPOlogy repo](https://github.com/todogroup/ospology/discussions), indicating the desired location and how the creation of this meet-up can benefit the OSPO Community. The [TODO Steering Committee](https://github.com/todogroup/governance#-about-todo-steering-committee-tsc) will review this proposal and has the final word to approve/decline.
```
Comment (geekygirldawn) [inline local-meetups/framework.md:72]: Again, I'd remove the hyphen between meet and up

**Review by geekygirldawn — CHANGES_REQUESTED** [review]
This looks great, thanks for doing this! I made mostly a few minor wording suggestions.

**Comment by anajsana** [comment]
@geekygirldawn Thank you! committed changes with the requested additions 👍

**Comment by anajsana** [comment]
Changes requested by the SC were added to the framework. Merging this PR to move forward. In case something else needs to be added or changed, please open a new PR.

**Pull request merged**